src/js/views/patients/actions/actions_views.js

```diff
@@ -254,6 +254,80 @@ const DueComponent = Component.extend({
   },
 });
 
+// Every 15 mins for 24 hours starting at 7am
+const start = moment({ hour: 6, minute: 45, second: 0 });
+
+const times = _.times(96, function() {
+  return { time: start.add(15, 'minutes').format('h:mm A') };
```

Comment:
I think this format should actually be `LT` for i18n consideration.. here and below where it reads the format.